packages/shared/src/features/onboarding/shared/FunnelBannerMessage.tsx

```diff
@@ -0,0 +1,42 @@
+import type { ComponentProps } from 'react';
+import React, { useMemo } from 'react';
+import classNames from 'classnames';
+import { LazyImage } from '../../../components/LazyImage';
+import { sanitizeMessage } from './utils';
+import type { FunnelBannerMessage } from '../types/funnel';
+
+type FunnelStepBannerProps = FunnelBannerMessage & ComponentProps<'div'>;
+
+export const FunnelBanner = ({
```

Comment:
Maybe we want to rename the file too, wdyt? @sshanzel

Reply:
Right. To make them consistent. I'll update it 🚀